main/Motor.cpp

```diff
@@ -132,3 +131,36 @@ void Motor::coast()
     ledcWrite(CHANNEL_2, 0);
     ledcWrite(CHANNEL_3, 0);
 }
+
+void Motor::debug_show_command(BlockModel blockModel)
+{
+    int state = blockModel.get_block_state();
+    float time = blockModel.get_time() * 0.1; // get_time()が10で1秒
+
+    switch(state)
+    {
+        case 1:
+            Serial.print("前進 を ");
```

Comment:
consoleで表示するとずれるよね．デバッグメッセージは半角で必ず等幅になる英語のほうが良いと思う．
![image](https://user-images.githubusercontent.com/12444722/47384957-b092ee00-d743-11e8-84bf-1c6b8657e7c1.png)
```suggestion
            Serial.print("Exec front ");
```